### ISSUE TITLE
Unable to login to the drupal portal 

### ISSUE THREAD

**Issue by rmeda24** [apigee/apigee-devportal-kickstart-drupal#465]
Hi, None of the users are able to login to the portal other than the admin user.

**Comment by shishir-intelli** [comment]
Hi @rmeda24,

Sorry for the late response. Can you have the logged in admin run developer sync from:
**admin/config/apigee-edge/developer-settings/sync**

Here's the documentation: https://www.drupal.org/docs/8/modules/apigee-edge/synchronize-developers-with-apigee-edge

Once the developer sync is complete, try logging in and let us know if this resolved the issue.

**Comment by shishir-intelli** [comment]
The issue was resolved internally. We are closing this issue. If you run into any problems, feel free to re-open this ticket or a new one (if the new issue is unrelated to this one). Thanks
